tool/tctl/common/auth_command.go

```diff
@@ -680,12 +680,25 @@ func (a *AuthCommand) generateUserKeys(ctx context.Context, clusterAPI auth.Clie
 	}
 	key.TrustedCA = auth.AuthoritiesToTrustedCerts(hostCAs)
 
+	kubeTLSServerName := ""
+	response, err := clusterAPI.Ping(ctx)
```

Comment:
@AntonAM is there any special reason why the `ProxyListenerMode` was added to the `PingResponse`?
I am asking because `clusterAPI` has direct access to `GetClusterNetworkingConfig` request which is exactly the only info you extract from the Ping req.

Reply:
Thanks for the info! I was advised to add it to ping, I'll check if it's still beneficial to have this info in our ping response 👍

Reply:
Yeah, I originally suggested placing this into ping response but since it's also available via API, I think it makes sense to just use that. Since we already have an authenticated client in tctl auth sign.

Reply:
I rolled back changes to the `Ping` command and we're now using `GetClusterNetworkingConfig` 👍